addons/pointwatch/message_ids.lua

```diff
@@ -211,7 +215,21 @@ local function update_offset(zone_id)
         end
 
         m.offset = res[1]
-        m.name = nil
+        m.update = nil
+        local s = {}
+        local n = 0
+        for zone, t in pairs(messages) do
+            if not t.update then
+                n = n + 1
+                s[n] = t.name .. '\t' .. tostring(t.offset)
+            end
+        end
+        if n > 0 then
```

Comment:
Wouldn't `n` always be `> 0`? At least `1`, because you set `m.update` to `nil` above, so if you iterate the messages after that, at least the current message (`m`) would result in `not t.update` being `true`?